libsolidity/ast/Types.cpp

```diff
@@ -865,32 +865,20 @@ bool RationalNumberType::isImplicitlyConvertibleTo(Type const& _convertTo) const
 		IntegerType const& targetType = dynamic_cast<IntegerType const&>(_convertTo);
 		unsigned forSignBit = (targetType.isSigned() ? 1 : 0);
 		if (m_value > rational(0))
-		{
-			if (m_value.numerator() <= (u256(-1) >> (256 - targetType.numBits() + forSignBit)))
-				return true;
-		}
-		else if (targetType.isSigned() && -m_value.numerator() <= (u256(1) << (targetType.numBits() - forSignBit)))
-			return true;
-		return false;
+			return m_value.numerator() <= (u256(-1) >> (256 - targetType.numBits() + forSignBit)) ? true : false;
```

Comment:
Please remove the ` ? true : false`, I think it complicates it and makes it harder to read (same below).